### PR TITLE
fix: Properly treat non-movers in EDGT Lyon ADISP data

### PR DESCRIPTION
**Pull request by LeonardoLuquezi**
* filter edgt lyon adisp data
* update edgt lyon adisp feature

### CODE REVIEW

**Comment by sebhoerl** [comment]
Thanks! Somehow this looks familar. This is something that was already fixed for Nantes, isn't it? I'll merge as soon as #166 goes through.

**Comment by LeonardoLuquezi** [comment]
Yes, that's right. It's the same treatment we do with the data from Nantes.

File: data/hts/edgt_lyon/cleaned_adisp.py [hunk 1]
```diff
@@ -163,7 +172,11 @@ def execute(context):
 
     # Chain length
     df_count = df_trips[["person_id"]].groupby("person_id").size().reset_index(name = "number_of_trips")
+    # People with at least one trip (number_of_trips > 0)
     df_persons = pd.merge(df_persons, df_count, on = "person_id", how = "left")
+    # People that awnsered the travel questionary section but stayed at home (number_of_trips = 0)
```
Comment (sebhoerl) [inline data/hts/edgt_lyon/cleaned_adisp.py:177]: awnsered > answered

**Review by sebhoerl — CHANGES_REQUESTED** [review]
Please add a note on top of `CHANGELOG.md`:

> Properly treat non-movers in EDGT Lyon ADISP data

**Review by sebhoerl — APPROVED** [review]

**Comment by sebhoerl** [comment]
Thansk a lot!

**Pull request merged**